web/src/message_reminder.ts

```diff
@@ -1,11 +1,18 @@
 import $ from "jquery";
+import type {z} from "zod";
 
 import * as channel from "./channel.ts";
 import * as feedback_widget from "./feedback_widget.ts";
 import {$t} from "./i18n.ts";
+import type {StateData, reminder_schema} from "./state_data.ts";
 import * as timerender from "./timerender.ts";
 import * as ui_report from "./ui_report.ts";
 
+export type Reminder = z.infer<typeof reminder_schema>;
+
+// reminders_data is a map where key=reminder id and value=reminders
+export const reminders_data = new Map<number, Reminder>();
```

Comment:
Probably calling this `reminders_by_id` would let you remove the comment; the type annotation tells you have the value type.

Reply:
Done